bgpd/bgp_zebra.c

```diff
@@ -396,12 +398,23 @@ static int bgp_interface_address_delete(int command, struct zclient *zclient,
 			   ifc->ifp->name, buf);
 	}
 
+	/* Update the router id if the interface address being deleted is
```

Comment:
Is this function called for IP address delete for both normal interface & Loopback interfaces ?